### PR TITLE
[Merged by Bors] - doc(Algebra/Category/Ring/Basic): fix copy-paste errors

### PR DESCRIPTION
**Pull request by eric-wieser**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
I haven't checked if the doc-strings are out of date in other way --- but these fixes certainly look correct. Thanks!
maintainer merge

**Comment by ocfnash** [comment]
bors merge

File: Mathlib/Algebra/Category/Ring/Basic.lean [hunk 1]
```diff
@@ -41,7 +41,7 @@ instance : CoeSort (SemiRingCat) (Type u) :=
 attribute [coe] SemiRingCat.carrier
 
 /-- The object in the category of R-algebras associated to a type equipped with the appropriate
-typeclasses. This is the preferred way to construct a term of `AlgebraCat R`. -/
+typeclasses. This is the preferred way to construct a term of `SemiRingCat`. -/
 abbrev of (R : Type u) [Semiring R] : SemiRingCat :=
```
Comment (alreadydone) [inline Mathlib/Algebra/Category/Ring/Basic.lean:45]: Why is this not SemiringCat?

Comment (ocfnash) [reply Mathlib/Algebra/Category/Ring/Basic.lean:45]: I think it should be!

**Pull request closed without merging**